Tests/UI/BillingForm/ViewModel/CardCreationModelTests.swift

```diff
@@ -85,4 +85,44 @@ final class CardCreationModelTests: XCTestCase {
         XCTAssertEqual(card?.billingAddress, model.billingAddress)
         XCTAssertEqual(card?.phone, model.phone)
     }
+    
+    func testCVVOptionalTrue() {
```

Comment:
Can you rename the function to be more meaningful having Suffix True is minimal I feel.